cmd/site-controller/deploy-watch-all-ns.yaml

```diff
@@ -86,6 +86,20 @@ rules:
   - create
   - delete
   - update
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterrolebindings
+  verbs:
+  - create
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterroles
+  verbs:
+  - bind
+  resourceNames:
+  - skupper-service-controller
```

Comment:
This is not really a comment on this PR, but I missed it earlier and now I've been looking at this more closely it seemed worth suggesting: is there a better name for the role? E.g. skupper-policy-reader?

Reply:
If you find it clearer, I will do that in a separate PR.

Reply:
I think it is better if the name describes what the role allows. Makes people less nervous about binding that role. Its a minor thing, but if you get a chance to do a separate PR with the change I think it would be a small but significant improvement.

Reply:
I agree. I will raise an issue and work on it. Thanks, I think it makes it clearer for sure.

Reply:
Created #706 to track this.